utils.py

```diff
@@ -0,0 +1,24 @@
+
+
+browsers_definition = {
+  "Chrome": {
+    "executable": "Google Chrome",
+    "identifier": "com.google.Chrome"
+  },
+  "Canary": {
+    "executable": "Google Chrome Canary",
+    "identifier": "com.google.Chrome.canary"
+  },
+  "Chromium": {
+    "executable": "Chromium",
+    "identifier": "org.chromium.Chromium"
+  },
+  "Edge": {
+    "executable": "Microsoft Edge",
+    "identifier": "" # TODO: figure out Edge identifier
+  },
+  "Safari": {
+    "executable": "Safari",
+    "identifier": "com.apple.Safari"
```

Comment:
If this is used in any way to grab numbers from the plist this will need to be updated to include the Safari supporting processes like the SafeBrowsing or history processes.

Reply:
I don't think I'm using it to filter anymore (that was the case at some point)
Might want to keep it around since bundle name/identifier are the 2 ways applications can be identified by "open" on mac.